desktop.py

```diff
@@ -215,13 +221,14 @@ def run_main(raw_args, target_platform):
     actions = OrderedDict()
     actions['configure'] = configure
     actions['make'] = make
+    actions['copy_bcl'] = copy_bcl
```

Comment:
Action names should use a hyphen `-` instead of an underscore `_` to separate words.